life_expectancy/main.py

```diff
@@ -3,31 +3,36 @@
 
 import pandas as pd
 
-from life_expectancy.cleaning import clean_data
-from life_expectancy.loading_saving import load_data, save_data
+from life_expectancy.cleaning import DataCleaner
+from life_expectancy.loading_saving import DataLoader, save_data
 
 PARENT_PATH = Path(__file__).parent
 FILE_PATH = PARENT_PATH / 'data'
-INPUT_FILE_NAME = 'eu_life_expectancy_raw.tsv'
 OUTPUT_FILE_NAME = 'pt_life_expectancy.csv'
-INPUT_FILE_PATH = FILE_PATH / INPUT_FILE_NAME
 OUTPUT_FILE_PATH = FILE_PATH / OUTPUT_FILE_NAME
```

Comment:
These global contants are also being define on loading_saving. If, in the future you need to change them, but forget to change both, you will have errors. Try to remove this duplication by either passing the value as parameters or making these modules import the same constant

Reply:
Thank you for your comment @MrCordeiro! I just changed this after your comment in PDS. 
Which variable is duplicated?

Reply:
Oh, you've since removed them! Great!